packages/grafana-schema/src/common/data_frame.cue

```diff
@@ -0,0 +1,216 @@
+package common
+
+//  See also:
+//  https://github.com/grafana/grafana-plugin-sdk-go/blob/main/data/frame_type.go
+DataFrameType:
+		"timeseries-wide" |
+		"timeseries-long" |
+		 // @deprecated in favor of TimeSeriesMulti
```

Comment:
Looks like comments on enum fields don't work as they are not present in the generated types.